docs/buildah-bud.md

```diff
@@ -42,6 +42,10 @@ Add an image *annotation* (e.g. annotation=*value*) to the image metadata. Can b
 
 Note: this information is not present in Docker image formats, so it is discarded when writing images in Docker formats.
 
+**--arch**="ARCH"
+
+Prefer ARCH instead of the architecture of the machine when building images.
```

Comment:
* s/Prefer/Set/
* s/when building images/in the image manifest and config/

Reply:
Fixed :+1: